tests/test_functions.py

```diff
@@ -43,11 +45,21 @@ def test_matrix_correlation(self):
         FC = func.fc(self.model.BOLD.BOLD)
         cc = func.matrix_correlation(FC, self.ds.FCs[0])
 
+    def test_weighted_correlation(self):
+        x = self.model.rates_exc[0, :]
+        y = self.model.rates_exc[1, :]
+        w = np.ones(x.shape)
+        cc = func.weighted_correlation(x, y, w)
```

Comment:
no numerical comparison then? so just testing whether it runs?

Reply:
Mhh yes that's it for now. Should I add synthetic data and testing or you think its ok?

Reply:
naaah, it's ok.. after all these are **unit** tests (whether shit runs) and not **functional** tests